### PR TITLE
Adds Arquillian Cube 1.0.0 announcement

### PR DESCRIPTION
**Pull request by lordofthejars**

### CODE REVIEW

File: blog/arquillian-cube-1.0.0.textile [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+date: 2017-03-24
+author: lordofthejars
+---
+
+p. New version of Arquillian Cube 1.0.0 has been released with one new feature and few bug fixes.
```
Comment (bartoszmajsak) [inline blog/arquillian-cube-1.0.0.textile]: Would be good to also explain that we switch versioning. Personally I would also add some excitement paragraph that we are finally out of never ending alpha :)

Comment (lordofthejars) [inline blog/arquillian-cube-1.0.0.textile]: But semantic version is something globally, should not be mentioned there instead of in each module?

Comment (bartoszmajsak) [inline blog/arquillian-cube-1.0.0.textile]: It was my intention to do it globally, but we started rolling it per module basis, so maybe it's worth to mention?

**Comment by bartoszmajsak** [comment]
Upstream @59018c357d51b4404cc2187eec156cec9a0a1e9a

**Pull request closed without merging**